packages/grafana-schema/src/common/data_frame.cue

```diff
@@ -0,0 +1,216 @@
+package common
+
+//  See also:
+//  https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/frame_type.go
+DataFrameType:
+		"timeseries-wide" |
+		"timeseries-long" |
+		 // @deprecated in favor of TimeSeriesMulti
+		"timeseries-many" |
+		"timeseries-multi" |
+		// Directory listing
+		"directory-listing" |
+		// First field is X, the rest are ordinal values used as rows in the heatmap
+		"heatmap-rows" |
```

Comment:
Generating this fails without providing `memberNames`, looks like an issue with enum keys that contain dashes.